app/server/routes/api.ts

```diff
@@ -91,11 +91,36 @@ router.post(
   )("direct-debit/check-account", true)
 );
 
-router.post("/case", sfCasesApiHandler("case"));
+router.use(
+  "/case/:caseId?",
+  apiGatewayHandler(
+    "cancellation-sf-cases-api",
+    pathParams => `/case/${pathParams.caseId || ""}`
+  )
+);
 
-router.patch(
-  "/case/:caseId",
-  sfCasesApiHandler("case/:caseId", false, "caseId")
+router.use(
+  "/holidays/:subscriptionName?/:sfId?",
```

Comment:
Is it necessary to have logic in the router?

Reply:
eliminated in https://github.com/guardian/manage-frontend/pull/373#discussion_r396399572

Reply:
eliminated in https://github.com/guardian/manage-frontend/pull/373/files#r396404506

Reply:
eliminated in https://github.com/guardian/manage-frontend/pull/373#discussion_r396404506